### PR TITLE
Rockfield: Fix cart button styles

### PR DESCRIPTION
**Pull request by ouikhuan**
#### Changes proposed in this Pull Request:

This PR fixes the button styles in the Cart page for Proceed to checkout button and also remove the outline from the remove item link.

Before:

Screenshot 2024-11-28 at 6 21 47 PM

After:

Screenshot 2024-12-06 at 2 33 23 PM

#### Related issue(s):
#8467

### CODE REVIEW

**Comment by ouikhuan** [comment]
Hi @Automattic/theam 👋 I'm working on this issue from GroundsKeeping rotation so first timer here! The PR is a CSS fix, but from the theme check above it seems to trigger quite a few error that's not related to the proposed fix, what would you suggest for us to move this forward?

**Comment by alaczek** [comment]
Could we style this button to match the theme?

image

I had a quick look at the failing checks, and I don't think we have to worry about this - I think they are looking for theme.json which this theme doesn't have as it's older.

**Comment by ouikhuan** [comment]
> I had a quick look at the failing checks, and I don't think we have to worry about this - I think they are looking for theme.json which this theme doesn't have as it's older.

Thanks for having a look @alaczek !

> Could we style this button to match the theme?

Sure, I've updated the CSS so it's matching the theme style now and ready for re-review, could you have another look once you have some time? Thank you!

Screenshot 2024-12-06 at 2 33 23 PM

**Review by alaczek — APPROVED** [review]
This is looking good to me now. LGTM!

File: rockfield/style-rtl.css [hunk 1]
```diff
@@ -2850,7 +2850,7 @@ body:not(.fse-enabled) .site-description {
 	z-index: 1;
 }
 
-.main-navigation > div > ul li:hover, .main-navigation > div > ul li[focus-within] {
+.main-navigation > div > ul li:hover, .main-navigation.js-focus-within > div > ul li[focus-within], .js-focus-within .main-navigation > div > ul li[focus-within] {
```
Comment (scruffian) [inline rockfield/style-rtl.css:2853]: This PR changed some selectors accidentally I believe (see https://github.com/Automattic/themes/pull/8570)

**Pull request merged**